dynamic-plugins/imports/package.json

```diff
@@ -18,7 +18,7 @@
     "@janus-idp/backstage-plugin-acr": "1.2.4",
     "@janus-idp/backstage-plugin-jfrog-artifactory": "1.2.4",
     "@janus-idp/backstage-plugin-nexus-repository-manager": "1.4.4",
-    "@janus-idp/backstage-plugin-ocm": "3.3.4",
+    "@janus-idp/backstage-plugin-ocm": "3.5.0",
```

Comment:
Please let this be updated by #569

This version update is irrelevant to the PR merit.